packages/web-components/package.json

```diff
@@ -96,11 +98,16 @@
     "e2e:local": "node ./scripts/e2e.js --ui"
   },
   "devDependencies": {
+    "@custom-elements-manifest/analyzer": "^0.10.4",
```

Comment:
please move all of these to root package.json to follow single version policy, ty

Reply:
Are you sure? These packages are very specific to web components.

Reply:
@break-stuff yeah, it's part of our one-version policy to have packages at the root of the monorepo as well.

Reply:
Updated